rwengine/src/audio/SoundSource.cpp

```diff
@@ -248,7 +248,7 @@ struct InputData {
 /// to buffer.
 static int read_packet(void* opaque, uint8_t* buf, int buf_size) {
     auto* input = reinterpret_cast<InputData*>(opaque);
-    buf_size = FFMIN(buf_size, input->size);
+    buf_size = FFMIN(buf_size, static_cast<int>(input->size));
```

Comment:
Here can be used `std::fmin` instead of this ugly macro. IIRC it's clang warn.

Reply:
`std::fmin` is for floating points. That would create new warnings.
`std::min` has only templates for arguments of the same type.